includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -42,6 +42,12 @@ public function enforce_sizes_attribute( $attributes ) {
 
 		$attributes['sizes'] = sprintf( '(min-width: %1$dpx) %1$dpx, 100vw', absint( $max_width ) );
 
+		// Convert attributes like width="300px" to width="300"
+		$attributes['width'] = absint( $attributes['width'] );
```

Comment:
Should we do this higher up in the method? We use `width` as the `max_width` value earlier and that could benefit from the `absint`.

One other issue is percentage values like `80%`. Passing that through `absint` converts the value to `80` which is not actually what we want.